src/main/java/org/kitteh/irc/client/library/implementation/EventListener.java

```diff
@@ -131,6 +132,7 @@ public void welcome(ClientReceiveNumericEvent event) {
     @Handler(priority = Integer.MAX_VALUE - 1)
     public void version(ClientReceiveNumericEvent event) {
         this.client.resetServerInfo();
+        boolean isTwitch = this.client.getEventManager().getRegisteredEventListeners().stream().anyMatch(listener -> (listener instanceof TwitchListener));
```

Comment:
This is purely a convenience action to make life easier for some folks. By the time I get to letting some of the bigger pieces get swapped out, there will be major rewrites to the EventListener class anyway. :)